### DISCUSSION TITLE
Add utility for 'overflow-anchor: none'

### DISCUSSION THREAD

**Discussion by kripod** [tailwindlabs/tailwindcss#2301] [Ideas]
Sometimes, [scroll anchoring](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-anchor) can be destructive (e.g. on `contenteditable` HTML elements with a padding). I would like to propose adding a `no-overflow-anchoring` utility for setting `overflow-anchor: none` easily.

**Comment by RobinSCU** [comment]
I also need this for our webshop

**Comment by kripod** [comment]
@RobinSCU In the meantime, you may use a `tailwind.config.js` file like below:

```js
const plugin = require("tailwindcss/plugin");

module.exports = {
  plugins: [
    plugin(({ addUtilities }) => {
      addUtilities({
        ".no-overflow-anchoring": {
          overflowAnchor: "none",
        },
      });
    }),
  ],
};

```

**Reply by RobinSCU** [reply]
@kripod Thanks!

**Reply by zanona** [reply]
Also as an [arbitrary property](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties), for example:
```jsx
<div className="bg-red-500 md:[overflow-anchor:none]">
...
</div>
```

**Reply by JoaoPedroMDP** [reply]
Thank you my savior

**Comment by Zerebokep** [comment]
any chance to get this added?

**Comment by programming-with-ia** [comment]
+1

**Comment by orimay** [comment]
+1

**Comment by switz** [comment]
Would be very nice!

**Comment by SlothLady** [comment]
+1

**Comment by rozsazoltan** [comment]
From v4 onwards, this can easily be done using the `@utility` directive:

```css
@utility no-overflow-anchoring {
  overflow-anchor: none;
}
```

For v3 just follow https://github.com/tailwindlabs/tailwindcss/discussions/2301#discussioncomment-60981.

Or can use arbitrary value:
```html
<div class="[overflow-anchor:none]">...</div>
<div class="lg:[overflow-anchor:none]">...</div>
```

**Reply by SlothLady** [reply]
Thank you!
